src/strings/en-us.json

```diff
@@ -928,6 +928,12 @@
     "MediaInfoSampleRate": "Sample rate",
     "MediaInfoSize": "Size",
     "MediaInfoTimestamp": "Timestamp",
+    "MediaInfoSoftware": "Software",
+    "MediaInfoStreamTypeAudio": "Audio",
+    "MediaInfoStreamTypeData": "Data",
+    "MediaInfoStreamTypeEmbeddedImage": "Embedded Image",
+    "MediaInfoStreamTypeSubtitle": "Subtitle",
+    "MediaInfoStreamTypeVideo": "Video",
```

Comment:
can you find when this was removed?..

Reply:
https://github.com/jellyfin/jellyfin-web/commit/381ee59b9eaaa7dfa0c05713daa5a9288d8ec86b